src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+import './Calculator.scss';
+
+function Calculator() {
+  return (
+    <section id="calculator">
+      <table>
+        <tr className="result">
+          <td>0</td>
+        </tr>
+        <tr className="buttons-wrap">
```

Comment:
- [ ] @dennis-every kindly check how you structure your view, I got the error below when running the app 
![image](https://user-images.githubusercontent.com/61408860/218826736-66048aec-695d-49d5-88a5-ed7a42b26c60.png)